styles/pages/mainmenu.scss

```diff
@@ -43,8 +44,8 @@ $drawer-width: 768px;
 	height: 100%;
 
 	&__container {
-		width: 1920px;
-		height: 1080px;
+		width: 91%; // counteract blur 110% size
+		height: 91%; // counteract blur 110% size
```

Comment:
This just feels so wrong. But it's completely my fault, the >100% scaling of the blur container was stupid of me. I'd like to hop onto this branch redo the scaling of the blur container and this, if that's okay with you

Reply:
Yea, that's fine with me. I couldn't figure out how to prevent the right edge of the screen from miscalculating the blur without this (or an explicit size in pixels).